corehq/apps/app_manager/views/apps.py

```diff
@@ -425,6 +425,10 @@ def copy_app(request, domain):
         with zipfile.ZipFile(gzip, 'r', zipfile.ZIP_DEFLATED) as z:
             source = z.read(z.filelist[0].filename)
         app_id_or_source = source
+    elif request.POST.get('build_id'):
+        build = Application.get(request.POST.get('build_id'))
```

Comment:
nit: `get_app` is better?

Reply:
Is there a difference?

Reply:
Just correctly wraps the class, if needed and confirms the domain of the app.

Reply:
good point - https://github.com/dimagi/commcare-hq/pull/24943